plugins/package/src/main/kotlin/net/corda/cli/plugins/packaging/CreateCpb.kt

```diff
@@ -0,0 +1,130 @@
+package net.corda.cli.plugins.packaging
+
+import java.nio.file.Files
+import java.nio.file.Path
+import java.util.UUID
+import picocli.CommandLine.Command
+import picocli.CommandLine.Option
+import java.nio.file.StandardOpenOption.WRITE
+import java.nio.file.StandardOpenOption.READ
+import java.util.jar.Attributes
+import java.util.jar.JarEntry
+import java.util.jar.JarOutputStream
+import java.util.jar.Manifest
+import net.corda.cli.plugins.packaging.signing.CpxSigner
+
+@Command(
+    name = "create-cpb",
+    description = ["Creates a CPB from passed in CPK archives."]
+)
+class CreateCpb : Runnable {
+
+    @Option(names = ["--cpks"], required = true, split = " ", description = ["CPKs to convert into CPB"])
+    lateinit var cpks: List<String>
+
+    @Option(names = ["--cpb-name"], required = true, description = ["Manifest attribute CPB name"])
+    lateinit var cpbName: String
+
+    @Option(names = ["--cpb-version"], required = true, description = ["Manifest attribute CPB version"])
+    lateinit var cpbVersion: String
+
+    // Is the below option required? If not what should be its default value?
+    @Option(names = ["--upgrade"], required = true, description = ["CPB Upgrade option"])
+    lateinit var cpbUpgrade: String
```

Comment:
The answer to the question is no, it's not required. It should default to `false`.

Can we change the description to "Allow upgrade without flow draining" please?

Reply:
Done